### ISSUE TITLE
Couldn't find a link from blockchain/ to lockchain/public-private-keys/

### ISSUE THREAD

**Issue by onqtam** [anders94/blockchain-demo#46]
I think it should be easier for people in https://anders.com/blockchain/ to reach https://anders.com/blockchain/public-private-keys/

**Comment by anders94** [comment]
Agreed - currently you can click the link above the 2nd video on https://anders.com/blockchain/ but where would you suggest I also link?
